src/coreclr/jit/CMakeLists.txt

```
@@ -301,8 +301,8 @@ set( JIT_HEADERS
   host.h
   hostallocator.h
   hwintrinsic.h
-  ICorJitInfo_API_names.h
-  ICorJitInfo_API_wrapper.hpp
+  ICorJitInfo_API_names_generated.h
+  ICorJitInfo_API_wrapper_generated.hpp
```

```suggestion
  ICorJitInfo_names_generated.h
  ICorJitInfo_wrapper_generated.hpp
```
Drop `API` from the name to make it a bit shorter?